src/components/Logbar/Logbar.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import { BsPersonCircle } from "react-icons/bs";
+import { BsPencilSquare } from "react-icons/bs";
+import { BsLockFill } from "react-icons/bs";
+
+import "./Logbar.scss";
+
+const Logbar = () => {
+  const [selectedOption, setSelectedOption] = useState("login");
+
+  const handleOptionChange = (event) => {
+    setSelectedOption(event.target.value);
+  };
+
+  return (
+    <div className="logBarMainContainer">
+      <label>
+        <input
+          type="radio"
+          name="option"
+          value="login"
+          checked={selectedOption === "login"}
+          onChange={handleOptionChange}
+        />
+        <span className="checkmark">
+          <BsPersonCircle className="logIcons"/>
+          Login
+        </span>
+      </label>
+      <label>
+        <input
+          type="radio"
+          name="option"
+          value="register"
+          checked={selectedOption === "register"}
+          onChange={handleOptionChange}
+        />
+        <span className="checkmark">
```

Comment:
same here, add common class for 3 checkmarks but describe where it belongs. Apply same styles to 3 spans like:
checkmark-login, checkmark-register, checkmark-restorePass (for example)